src/mlpack/tests/main_tests/linear_regression_test.cpp

```diff
@@ -41,19 +46,13 @@ struct LRTestFixture
   }
 };
 
-void ResetSettings()
-{
-  IO::ClearSettings();
-  IO::RestoreSettings(testName);
-}
```

Comment:
I don't get why this had to move; did it?  Maybe I missed something about catch?

Reply:
Umm, not needed, I just packed it it into the structure , Should I change it

Reply:
I think it's better to leave it outside the structure if you are okay with that. :+1: